domain-services/fiscal-event-post-processor/druid-ingestion-config.json

```diff
@@ -155,6 +155,8 @@
           "attributes.departmentEntity.ancestry[10].name",
           "attributes.departmentEntity.ancestry[10].hierarchyLevel",
 
+          "attributes.accountNumber",
```

Comment:
Rename this to PSPCL account number. In future, there might other account numbers too.